file_test.go

```diff
@@ -52,6 +52,58 @@ func TestOpenFile(t *testing.T) {
 	}
 }
 
+func TestOpenFileWithoutPageIndex(t *testing.T) {
+	for _, path := range testdataFiles {
+		t.Run(path, func(t *testing.T) {
+			f, err := os.Open(path)
+			if err != nil {
+				t.Fatal(err)
+			}
+			defer f.Close()
+
+			s, err := f.Stat()
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			fileWithIndex, err := parquet.OpenFile(f, s.Size(), parquet.SkipPageIndex(true))
```

Comment:
Both `fileWithIndex` and `fileWithoutIndex` both skip reading the page index is that intentional?

Reply:
Good catch! This should be fixed now.